int32 personId
---
int32 status
int32 recognitionId
float32 similarityScore

#registration status constansts
int32 RECOGNITION_PASSED_PERSON_RECOGNIZED=0
int32 RECOGNITION_PASSED_PERSON_NOT_RECOGNIZED=1
int32 RECOGNITION_FAILED=2
int32 RECOGNITION_FAILED_FACE_NOT_DETECTED=3
int32 RECOGNITION_FAILED_FACE_NOT_CLEAR=4
int32 RECOGNITION_FAILED_PERSON_TOO_FAR=5
int32 RECOGNITION_FAILED_PERSON_TOO_CLOSE=6
int32 RECOGNITION_FAILED_FACE_AMBIGUITY=7
